source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -274,10 +275,14 @@ HttpConnectionManagerConfig::HttpConnectionManagerConfig(
     overall_sampling.set_numerator(
         tracing_config.has_overall_sampling() ? tracing_config.overall_sampling().value() : 100);
 
+    uint32_t max_path_tag_length = tracing_config.has_max_path_tag_length()
```

Comment:
Nit: suggest making this `const` and using https://github.com/envoyproxy/envoy/blob/master/source/common/protobuf/utility.h#L20.

Reply:
Done.